plugins/terraform/src/components/Terraform/Terraform.test.tsx

```diff
@@ -23,8 +24,8 @@ describe('Terraform', () => {
   });
 
   it('renders MissingAnnotationEmptyState when annotation is not present', async () => {
-    const missingAnnotation = { ...mockEntity };
-    // @ts-ignore
+    // If the following is refactored, ensure it is cloning mockEntity, and not merely referencing it!
+    const missingAnnotation = JSON.parse(JSON.stringify(mockEntity));
```

Comment:
Although the reference to the mockEntity object is a constant, it's properties are mutable.

Consequently, this test cleared out all its properties.  This was fine as this was the last test in the test suite.

However, any further re-use of the original mockEntity would fail as it was now empty.

It was necessary to clone the mockEntity in this way so that other tests are not affected.

Reply:
Ah I see the line below we were mutating a couple levels down.